app/src/main/res/layout/legend_row.xml

```diff
@@ -3,24 +3,25 @@
     android:orientation="vertical"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:background="@color/cellDefault"
     >
 
     <ImageView
         android:id="@+id/fragment_legend_image"
         android:layout_width="50dp"
         android:layout_height="50dp"
         android:src="@drawable/k"
-        />
+        xmlns:tools="http://schemas.android.com/tools"/>
     <TextView
         android:id="@+id/fragment_legend_name"
         android:layout_width="wrap_content"
         android:textSize="50sp"
         android:layout_height="wrap_content"
         android:layout_toEndOf="@id/fragment_legend_image"
-        android:text="knit"
+        android:text="@string/knit"
         android:paddingLeft="10dp"
+        xmlns:tools="http://schemas.android.com/tools"
```

Comment:
This line should go up in the root element. And then you can say `tools:text="@string/knit`, and it will only show up in the preview.